public-ui/src/app/data-browser/charts/chart-top-results/chart-top-results.component.ts

```diff
@@ -0,0 +1,79 @@
+import { Component, EventEmitter, Injector, OnChanges, Output } from '@angular/core';
```

Comment:
Chart-top-results extends chart-base 
also new naming convention for chart components